imports/client/components/FancyEditor.tsx

```diff
@@ -126,19 +128,36 @@ export const MentionSpan = styled.span<{
   display: inline-block;
   overflow-wrap: break-word;
   border-radius: 4px;
-  color: #4649ef;
-  background-color: #ced0ed;
+  color: ${({ theme }) => theme.colors.mentionSpanText};
+  background-color: ${({ theme }) => theme.colors.mentionSpanBackground};
   ${({ $isSelf }) =>
     $isSelf &&
     css`
-      background-color: #4649ef;
-      color: #fff;
+      background-color: ${({ theme }) => theme.colors.mentionSpanBackground};
+      color: ${({ theme }) => theme.colors.mentionSpanText};
     `}
   font-size: 0.8rem;
 `;
 
-const SelectedMentionSpan = styled(MentionSpan)`
-  box-shadow: 0 0 0 2px #b4d5ff;
+export const PuzzleSpan = styled.span<{
```

Comment:
This doesn't appear to be used in this patchset?  It's imported but not used in `ChatMessage.tsx`.  Maybe connected to some other feature?